README.md

```diff
@@ -2,4 +2,134 @@
+* patch or update applications or libraries included in the Flatcar OS image
+* add or remove applications and / or libraries
+* Modify the kernel configuration and add or remove kernel modules included with Flatcar
+* Build OS images for a variety of targets (qemu, bare metal, AWS, Azure, VMWare, etc.)
+* And lastly, the SDK can be used to upgrade SDK packages and to build new SDKs
+
 [flatcar-docs]: https://docs.flatcar-linux.org/os/sdk-modifying-flatcar/
+
+# Using the scripts repository: submodules and tags
+
+The repository is meant to be the entry point for Flatcar builds and development.
+For building pakages, there are 2 addigional repositories, [coreos-overlay](https://github.com/flatcar-linux/) and [portage-stable](https://github.com/flatcar-linux/portage-stable), which contain all packages' `ebuild` (build configuration) files.
+These repositories are included in `scripts` via git submodules and are used by the SDK container wrapper scripts detailed on further below.
+The submodules reside in:
+```
+scripts
+   +--sdk_container
+          +---------src
+                     +--third_party
+                             +------coreos-overlay
+                             +------portage-stable
+```
+
+When working with the scripts repo always make sure to initialise and to update these submodules; otherwise builds will break because build configuration is missing:
+```bash
+$ git clone https://github.com/flatcar-linux/scripts.git
+$ cd scripts
+$ git submodule init
+$ git submodule update
+```
+
+The `scripts` repository makes ample use of tags to mark releases.
+Sometimes, local and origin tags can diverge (e.g. when re-tagging something locally to test a build).
+Also, `git pull` and `git fetch` do not automatically pull new tags, so long-standing local sourcetrees may lack newer versions.
+To fetch and update all tags and to remove tags locally which have been deleted upstream, do
+```
+$ git pull --all --tags --prune --prune-tags
+```
+If upstream retagged (of if a tag was changed locally) the corresponding upstream tag will not be pulled so the local tag remains.
+In order to override local tags with upstream, run
+```
+$ git pull --all --tags --prune --prune-tags --force
+```
+
+# Using the SDK container
+
+We provide a containerised SDK via https://github.com/orgs/flatcar-linux/packages. The container comes in 3 flavours:
+* Full SDK initialised with both architectures supported by Flatcar (amd64 and arm64). This is the largest container, it's about 8GB in size (~3 GB compressed).
+* AMD64 SDK initialised for building AMD64 OS images. About 6GB in size (2GB compressed).
+* ARM64 SDK initialised for building ARM64 OS images on AMD64 hosts. Also about 6GB in size.
+While work on a native ARM64 native SDK is ongoing, it's unfortunately not ready yet. If you want to help, patches are welcome!
+
+The container can be run in one of two ways - "standalone", or integrated with the [Scripts](https://github.com/flatcar-linux/scripts) repo:
```

Comment:
I'd write "scripts" instead of "Scripts". Use of the capitalization is not consistent later.

```suggestion
The container can be run in one of two ways - "standalone", or integrated with the [scripts](https://github.com/flatcar-linux/scripts) repo:
```